site/.vitepress/config.ts

```diff
@@ -31,7 +31,12 @@ export default defineConfig({
     ],
 
     search: {
-      provider: "local",
+      provider: "algolia",
+      options: {
+        appId: "IHLWJWIZ3J",
+        apiKey: "e27b06dcc3597664309c1ed9b8cb5470",
+        indexName: "aa-sdk-staging",
```

Comment:
should this be `aa-sdk` ?

Reply:
yeah these values are old. sorry left a comment on asana ticket. waiting for new values for the apikey and index from algolia via email.

Reply:
not planning to merge until this is in

Reply:
let's put these in .env if possible? Is it okay to share these app id/api key?

Reply:
This is fine because algolia is indexing our website only. If someone ripped our key, their search would be using our docs not theirs, so not in their interest.

Also, this service is free and not rate limited, so nbd. Viem exposes their fields too.